packages/labs/analyzer/src/lib/analyzer.ts

```diff
@@ -113,26 +121,51 @@ export class Analyzer {
       // The jsPath appears to come out of the ts API with unix
       // separators; since sourcePath uses OS separators, normalize
       // this so that all our model paths are OS-native
-      jsPath: path.normalize(jsPath) as PackagePath,
+      jsPath: absoluteToPackage(
+        path.normalize(jsPath) as AbsolutePath,
+        this.packageRoot
+      ),
       sourceFile,
     });
 
+    this.programContext.currentModule = module;
+
     for (const statement of sourceFile.statements) {
       if (ts.isClassDeclaration(statement)) {
-        if (isLitElement(statement, this.checker)) {
+        if (isLitElement(statement, this.programContext)) {
           module.declarations.push(
-            getLitElementDeclaration(statement, this.checker)
+            getLitElementDeclaration(statement, this.programContext)
           );
-        } else {
+        } else if (isExported(statement)) {
```

Comment:
Why is this check here?

Reply:
Under my (maybe wrong, unclear) assumption that the only things we should expose in the model are exported declarations (or registered custom elements, which technically don't need to be exported since they're globally registered. As Justin pointed out today, you could subclass a private, non-exported class and need a reference to it, so might need to revisit this when we get to subclassing and generating the CEM. Can add a TODO.